docker/manage

```diff
@@ -319,6 +255,69 @@ deleteVolumes() {
 toLower() {
   echo $(echo ${@} | tr '[:upper:]' '[:lower:]')
 }
+
+initializeUserPrompts() {
+  PS3="Is your agent single-tenant or multi-tenant? "
+  select opt in "Single-Tenant" "Multi-Tenant/Traction" "Quit"; do
+    case $REPLY in
+      1)
+        echo "AGENT_TENANT_MODE=single" > .env
+        echo "AGENT_WALLET_SEED=$(generateSeed vc-authn-oidc)" >> .env
+        echo "ACAPY_HOST=http://aca-py" >> .env
+        break
+        ;;
+      2)
+        read  -p "Please provide your tenant's Wallet ID:" MT_ACAPY_WALLET_ID
+        read  -p "Please provide your tenant's Wallet Key:" MT_ACAPY_WALLET_KEY
+        echo "AGENT_TENANT_MODE=multi" > .env
+        echo "MT_ACAPY_WALLET_ID=${MT_ACAPY_WALLET_ID}" >> .env
+        echo "MT_ACAPY_WALLET_KEY=${MT_ACAPY_WALLET_KEY}" >> .env
+        echo "ACAPY_HOST=http://ngrok-traction-agent" >> .env
+        echo "AGENT_ADMIN_PORT=8080" >> .env
+        break
```

Comment:
Even using this option, the aca-py container is still started.

Reply:
Tested locally, aca-py and wallet containers do not get started when selecting multi-tenant/traction option. Can you try pulling the code and trying again (after running `./manage rm`)? It might have been something I pushed after you checked out the code.

Reply:
disregard, if the container exists, it will be restarted, if it is delete, ./manage correctly DOES NOT create the `aca-py` and `aca-py-ngrok` containers.